### PR TITLE
AB#896 Adjust admin msg panel

### PR DESCRIPTION
**Pull request by mkidawa**

### CODE REVIEW

**Review by ksobiecki — APPROVED** [review]
LGTM

File: src/common/components/Modals/MessageCreationModal/MessageCreationModal.tsx [hunk 1]
```diff
@@ -42,7 +44,13 @@ export const MessageCreationModal = () => {
             <CloseIcon onClick={handleClose} sx={{ cursor: 'pointer' }} />
           </Box>
         </Box>
-        Pozdrawiam wszystkich Polaków
+        <Box sx={{ pt: 2, px: 4, pl: 4 }}>
```
Comment (JKrymarys) [inline src/common/components/Modals/MessageCreationModal/MessageCreationModal.tsx:47]: If I recall properly `px` contains `pl`

File: src/layouts/PanelLayout/PanelLayout.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  logoColumnComponents?: ReactNode;
+  topBarComponents?: ReactNode;
+  children: ReactNode;
+}
+
+export const PanelLayout = ({ logoColumnComponents, topBarComponents, children }: PanelLayoutProps) => (
+  <Box sx={{ display: 'flex', width: '100vw', height: '100vh' }} component="section">
+    <LogoColumn>{logoColumnComponents}</LogoColumn>
+    <Box display="flex" flexDirection="column" width="100%">
+      {topBarComponents && <Box height="75px">{topBarComponents}</Box>}
```
Comment (JKrymarys) [inline src/layouts/PanelLayout/PanelLayout.tsx:15]: Are we sure to use pixels here?

**Review by JKrymarys — APPROVED** [review]

File: src/common/components/Modals/MessageCreationModal/MessageCreationModal.tsx (hunk 1 above)
Comment (mkidawa) [reply src/common/components/Modals/MessageCreationModal/MessageCreationModal.tsx:47]: Yeee, it was supposed to be pb instead of pl. Nice catch 😄

**Pull request merged**